includes/blocks/media-aside/render.php

```diff
@@ -20,11 +20,39 @@ function benenson_render_media_aside_block( array $attributes = [] ) {
 		$modal     = ! empty( $attributes['modal'] ) ? $attributes['modal'] : false;
 		$embed_url = ! empty( $attributes['embed'] ) ? $attributes['embed'] : '';
 		$video_url = ! empty( $attributes['videoId'] ) ? wp_get_attachment_url( $attributes['videoId'] ) : $embed_url;
+		$alignment = ! empty( $attributes['mediaAlignment'] ) ? $attributes['mediaAlignment'] : 'Left';
```

Comment:
On the original block, the media was aligned to the right. With the new alignment options, the original layout should be the default.
![](http://bigbite.im/q7SLlw+)